scss/civicrm/contact/pages/_import-contact.scss

```diff
@@ -21,7 +21,7 @@
   }
 
   #wizard-steps {
-    display: none;
+    margin: -30px -20px 0 -20px;
```

Comment:
leave a comment explaining what this is for, it's a very weird style

Reply:
we have the following HTML structure:

<div class="crm-block">
  <div id="wizard-steps">...</div>
  <h2>...</h2>
  <div class="help">...</div>
  ...other <div>'s with completely different classes
</div>

We have a white background color and padding on .crm-block so I am using a negative margin for the wizard steps in order to position it to match the .crm-block top, right and left borders.

Reply:
@mihaelmladenov I would need to understand this before we proceed on this margin